doc/source/whatsnew/v2.2.0.rst

```diff
@@ -351,6 +351,7 @@ Styler
 
 Other
 ^^^^^
+- Add complex128 to the types of numerical data we test across the test suite (:issue:`54761`)
```

Comment:
```suggestion
```

Not needed, unless we can describe a user facing behavior being changed

Reply:
OK.  I've deleted in my source and will commit with other potential changes in response to your comments.  No need to burn excess CI/CD cycles.